app/helpers/audits_helper.rb

```diff
@@ -237,7 +237,8 @@ def main_object?(audit)
     MAIN_OBJECTS.include?(type)
   end
 
-  def key_to_class(key)
-    @audit.auditable_type.constantize.reflect_on_association(key.sub(/_id(s?)$/, '\1'))&.klass
+  def key_to_class(key, audit = @audit)
```

Comment:
No need for the default here, it is being passed in from the `id_to_label` function